tmt/convert.py

```diff
@@ -8,10 +8,15 @@
 import re
 import subprocess
 from io import open
+from typing import Any, Dict, List, Optional, Tuple, Union, cast
 
 import fmf.utils
+# Used only for typing
+import nitrate as _nitrate
```

Comment:
Why the leading underscore? I suppose there's a reason for it, but it's not obvious, please add a comment.

Reply:
Wasn't needed after all, removed